examples/proc_FID_nutation.py

```diff
@@ -44,31 +44,31 @@
     s /= prscr.zeroth_order_ph(s["t2":0])
     s.ft("t2")
     fl.image(s, ax=ax1)
-    d_raw = s.C
     ax1.set_title("Signal pathway / ph0")
     # }}}
     # {{{ Look at phase variation
-    phase_var_s = s.C
-    d_uncorrected = phase_var_s.C.real.integrate("t2")
+    d_uncorrected = s.C.real.integrate("t2")
+    # PR again, you need to rename these
     for j in range(len(s.getaxis("beta"))):
         ph0 = prscr.zeroth_order_ph(phase_var_s["beta", j])
         phase_var_s["beta", j] /= ph0
     d_ind_ph0 = phase_var_s.real.integrate("t2")
     mysign = (d_ind_ph0 / d_uncorrected).angle / np.pi
     mysign = np.exp(1j * np.pi * mysign.run(np.round))
-    d_raw *= mysign
+    s *= mysign
```

Comment:
I just went ahead and made this change b/c I've asked for this many times, as well.